### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by JanWerder** [m1guelpf/yt-whisper#2]
The installation fails with the following error:

```
janwe@DESKTOP:~$ pip install git+https://github.com/m1guelpf/yt-whisper.git
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/m1guelpf/yt-whisper.git
  Cloning https://github.com/m1guelpf/yt-whisper.git to /tmp/pip-req-build-z4rcqme6
  Running command git clone --filter=blob:none --quiet https://github.com/m1guelpf/yt-whisper.git /tmp/pip-req-build-z4rcqme6
  Resolved https://github.com/m1guelpf/yt-whisper.git to commit 4e49b7851d8ee2f389f770b3e0967d4b1ac05a7d
  Preparing metadata (setup.py) ... done
Collecting whisper@ git+ssh://git@github.com/openai/whisper@main#egg=whisper
  Cloning ssh://****@github.com/openai/whisper (to revision main) to /tmp/pip-install-4woekdls/whisper_6de07a42a4564342bb35df700a95c428
  Running command git clone --filter=blob:none --quiet 'ssh://****@github.com/openai/whisper' /tmp/pip-install-4woekdls/whisper_6de07a42a4564342bb35df700a95c428
The authenticity of host 'github.com (140.82.121.3)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
  Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
  git@github.com: Permission denied (publickey).
  fatal: Konnte nicht vom Remote-Repository lesen.

  Bitte stellen Sie sicher, dass die korrekten Zugriffsberechtigungen bestehen
  und das Repository existiert.
  error: subprocess-exited-with-error

  × git clone --filter=blob:none --quiet 'ssh://****@github.com/openai/whisper' /tmp/pip-install-4woekdls/whisper_6de07a42a4564342bb35df700a95c428 did not run successfully.
  │ exit code: 128
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet 'ssh://****@github.com/openai/whisper' /tmp/pip-install-4woekdls/whisper_6de07a42a4564342bb35df700a95c428 did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by justincone-buck** [comment]
The relevant part of the error above is this: 

> Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
  git@github.com: Permission denied (publickey).
  
It sounds like you need to add your ssh key to Github. Try following these directions: 

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Good luck!

**Comment by m1guelpf** [comment]
Fixed by #9
